methods_exploration/it_functions.py

```diff
@@ -0,0 +1,99 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Tue Feb  8 10:11:52 2022
+
+@author: ggorski
+
+These functions were developed borrowing code and ideas from:
+    Laurel Larsen (laurel@berkeley.edu) for Larsen, L. G. and J. W. Harvey. 2017. Disrupted carbon cycling in restored and unrestored urban streams: Critical timescales and controls. Limnology and Oceanography, 62(Suppl. S1), S160-S182, doi: 10.1002/lno.10613. Please cite this work accordingly in products that use this code.
+    Edom Modges (edom.moges@berkeley.edu)
+    Julianne Quinn University of Virginia
+    The transfer entropy routine is based on Ruddell and Kumar, 2009, "Ecohydrologic process networks: 1. Identification," Water Resources Research, 45, doi:10.1029/2008WR007279.
```

Comment:
I like adding these credits to the READEME.md file in a section dedicated to references. You can also leave it here

Reply:
Yeah I think this is a good idea, right now I'm just listing my sources, but I like the idea of a more complete readme